ui/components/multichain/import-tokens-modal/import-tokens-modal.js

```diff
@@ -182,7 +182,9 @@ export const ImportTokensModal = ({ onClose }) => {
   );
   const selectedAccount = useSelector(getSelectedInternalAccount);
   const accounts = useSelector(getInternalAccounts);
-  const tokens = useSelector((state) => state.metamask.tokens);
+  const chainId = useSelector(getCurrentChainId);
+  const allTokens = useSelector((state) => state.metamask.allTokens);
```

Comment:
I wonder if this should be memoized into a `deepEqualSelector`.

Reply:
This should be fine as allTokens is directly from a controller store (we aren't doing any additional modifications.

Reply:
But you raised a good point, it might be nice to have a selector that allows us to select tokens for a single chain instead of selecting all tokens and then indexing on the client.